tests/integration/ovs_test.py

```diff
@@ -143,6 +143,22 @@ def test_vlan_as_ovs_bridge_slave(vlan_on_eth1):
         assertlib.assert_state_match(state)
 
 
+@pytest.mark.xfail(
+    raises=(NmstateLibnmError, AssertionError),
+    reason="https://bugzilla.redhat.com/1724901",
+)
+def test_ovs_interface_with_max_length_name():
+    bridge = Bridge(BRIDGE1)
+    ovs_interface_name = "ovs123456789012"
```

Comment:
Do we need a special test for this or could be just use this name in an existing test?

Reply:
There is only one such test and it already does several things. (`test_create_and_remove_ovs_bridge_with_internal_port_static_ip_and_mac`)

I do not mind using this long name there of you prefer, the only disadvantage is that it will overload it.